### ISSUE TITLE
training dataset size for transfer learning is 100 to 1000?

### ISSUE THREAD

**Issue by nyck33** [matterport/Mask_RCNN#1531]
When I ran mask-r-cnn on the tennis match video as is, it did not even detect the tennis ball at all.  Just the players and crowd mainly which was weird because there is a sports ball class.  

Still, I want to see if mask-r-cnn can track a tennis ball if I use transfer learning and set the config to just 2 classes:  background plus this new "ball" from the annotated image frames below.    

I am annnotating frames of a tennis broadcast per below:

  
![image](https://user-images.githubusercontent.com/25189545/58716578-8d27d400-83f3-11e9-81d4-c50162706546.png)

![image](https://user-images.githubusercontent.com/25189545/58716600-99139600-83f3-11e9-9719-996451230729.png)

![image](https://user-images.githubusercontent.com/25189545/58716638-a466c180-83f3-11e9-9fad-c795fb8fa532.png)

Note; this used Annotator version 2 but I have switched to version 1 as ver.2 is not compatible with the code as is.

**Comment by nyck33** [comment]
I was able to make some tweeks to the code to use the circular bounding shape but it looks inaccurate with only a training set size of 150 images and validation of 50 images.  

How many do I need of each to get the accuracy up?  

As you can see it thinks the ball is near the bottom left corner off the court:

![image](https://user-images.githubusercontent.com/25189545/58851961-d6f80f00-86be-11e9-9c04-c9bc6da61e89.png)

`ipynb` notebook and adjusted `balloon.py` are in my repo:  [repo](https://github.com/nyck33/tennisball-tracker/blob/master/tennisball_tracker_c_perimeter.ipynb)
